shared/domain/src/main/java/gov/va/vro/model/bip/BipUpdateClaimStatusResp.java

```diff
@@ -0,0 +1,18 @@
+package gov.va.vro.model.bip;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
```

Comment:
That's not necessary - required arguments are final and there is none of them here

Reply:
I think I copy/pasted many of them. Thanks for catching up. Removed.